runhouse/main.py

```diff
@@ -702,16 +716,25 @@ def cluster_list():
     table.add_column("Name", justify="left", no_wrap=True)
     table.add_column("Cluster Type", justify="center", no_wrap=True)
     table.add_column("Status", justify="left")
+    table.add_column("Last Active (UTC)", justify="left")
 
     # TODO: will be used if we'll need to print not-running clusters
     # all_clusters = running_clusters + not_running_clusters
 
     for rh_cluster in running_clusters:
-        table.add_row(
-            rh_cluster.get("Name"),
-            rh_cluster.get("Cluster Type"),
-            StatusColors.get_status_color(rh_cluster.get("Status")),
-        )
+        last_active_at = rh_cluster.get("Last Active (UTC)")
+
+        # Print Running clusters that were active it the last 24 hours
+        now = datetime.datetime.utcnow().replace(tzinfo=datetime.timezone.utc)
+        if (now - last_active_at).total_seconds() <= LAST_ACTIVE_AT_TIMEFRAME:
```

Comment:
wasnt so clear in the CLI output that only running clusters active in last 24 hrs are being shown, can we make that more clear in the table somehow?